app/views/project/edit.phtml

```diff
@@ -0,0 +1,61 @@
+<h3>Edit project</h3>
+
+{% set haveErrors = true %}
+{% if errors.filter('submit')|length == 1 and errors|length == 1 %}
+    {% set haveErrors = false %}
+{% endif %}
+
+{{ form('project/edit/' ~ project.id, 'method': 'post', 'id': 'project_edit_form') }}
```

Comment:
use the named routes with parameters where possible. Don't rely on concatenation when it comes to routes